wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/exception/ExceptionConstants.java

```diff
@@ -9,13 +9,15 @@ public class ExceptionConstants {
 
     public static ExceptionDataWrapper GETAWERTE_PARAMETER_UNVOLLSTAENDIG = new ExceptionDataWrapper("600",
             "getAWerte: Parameter unvollstaendig");
-
     public static ExceptionDataWrapper GETAWERTE_UNSAVEABLE = new ExceptionDataWrapper("601",
             "getAWerte: Die AWerte vom Client konnten nicht gespeichert werden.");
 
-    public static final ExceptionDataWrapper GET_STATUS_PARAMETER_UNVOLLSTAENDIG = new ExceptionDataWrapper("606", "getStatus: Parameter unvollstaendig");
-    public static final ExceptionDataWrapper POST_STATUS_PARAMETER_UNVOLLSTAENDIG = new ExceptionDataWrapper("607", "postStatus: Parameter unvollstaendig");
-    public static final ExceptionDataWrapper STATUS_UNSAVEABLE = new ExceptionDataWrapper("622", "postStatus: Der Status konnte nicht gespeichert werden.");
+    public static final ExceptionDataWrapper GET_STATUS_PARAMETER_UNVOLLSTAENDIG = new ExceptionDataWrapper("606",
+            "getStatus: Parameter unvollstaendig");
+    public static final ExceptionDataWrapper POST_STATUS_PARAMETER_UNVOLLSTAENDIG = new ExceptionDataWrapper("607",
+            "postStatus: Parameter unvollstaendig");
+    public static final ExceptionDataWrapper STATUS_UNSAVEABLE = new ExceptionDataWrapper("622",
+            "postStatus: Der Status konnte nicht gespeichert werden.");
```

Comment:
Diese Umformatierung gehört nicht in den PR

Reply:
Ich wollte das halt wegen der Leserlichkeit einheitlich machen. Würde ich mal so lassen.